client/CServerHandler.cpp

```diff
@@ -161,6 +162,19 @@ void CServerHandler::startLocalServerAndConnect()
 		threadRunLocalServer->join();
 
 	th->update();
+	
+	try
+	{
+		CConnection testConnection(settings["server"]["server"].String(), getDefaultPort(), NAME, uuid);
+		logNetwork->error("Port is busy, kill other vcmiserver processes");
+		CInfoWindow::showInfoDialog("Port is busy, kill other vcmiserver processes", CInfoWindow::TCompsInfo(), PlayerColor(1));
```

Comment:
would appreciate for ideas to make text more user-friendly

Reply:
- please move the hardcoded string into `config/translate.json`
- maybe something like: Another vcmiserver process is running, please terminate it first
- code improvement:
```suggestion
		CInfoWindow::showInfoDialog("Port is busy, kill other vcmiserver processes", {});
```

Reply:
Can we ask user if they want to terminate it and do it for them automatically? I think this way it will be the best. So I can try to do this myself. Is it possible to kill process via boost or whatever?

Reply:
> Is it possible to kill process via boost or whatever?

probably yes, but I'm pretty sure we'll need to pull in new boost module

Reply:
Or we can consider using single-process by default. This I like even more